packages/tofu/src/inspectSource.js

```diff
@@ -102,6 +104,11 @@ function inspectGlobals(
         if (path.length === 0) {
           return
         }
+        // Sometimes nesting further doesn't make sense, like in case of Function methods
+        // Eliminating cases of nesting unwanted path endings like func.bind.call was considered not worth the complexity
+        if (avoidEndingPathsIn.includes(path[path.length - 1])) {
+          path.pop()
+        }
```

Comment:
I don't understand what this does.

Reply:
If a path in generated policy ends with one of the items on the list, we pop that item off (mutate `path` in place.

so `document.getElementById.bind` becomes `document.getElementById`

Reply:
can you change the comment to be more clear about it?